docs/maintainers.md

```diff
@@ -1 +1,13 @@
-{!MAINTAINERS.md!}
+Trestle was designed and open sourced by a team based at [IBM Research](https://www.research.ibm.com/) and others around the world.  The list includes:
```

Comment:
This should not be changed. The mainters reference automatically reads the file at the root of the directory of the repository. That file should be changed instead.

Reply:
Fixed.